src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1362,18 +1364,21 @@ def add_units_table(
 
 def _add_waveforms_to_units_table(
     sorting: SortingExtractor,
-    units_table,
-    row_ids,
+    units_table: pynwb.misc.Units,
+    row_ids: List[int],
     skip_features: Optional[List[str]] = None,
 ):
     """
     Auxiliary method for adding waveforms to an existing units_table.
 
     Parameters
     ----------
-    sorting :  A spikeextractors SortingExtractor.
-    units_table : a previously created units table
-    skip_features : list of str
+    sorting : SortExtractor
```

Comment:
```suggestion
    sorting : SortingExtractor
```

Reply:
To match the explicit parent module used in other places, such as `pynwb.NWBFile`